Sources/Utility/PkgConfig.swift

```
@@ -282,12 +282,16 @@ struct PkgConfigParser {
         }
         
         var it = line.characters.makeIterator()
+        // Indicates if we're in a quoted fragment, we shouldn't append quote.
+        var inQuotes = false
         while let char = it.next() {
-            if char == "\\" {
+            if char == "\"" {
```

Ok, cool. Can we also throw an error on those cases then?